docs/backtesting.md

```diff
@@ -84,8 +84,8 @@ freqtrade backtesting --timerange=20190501-
 
```

Comment:
Is not it the same with that written below in the lines 85-88?
Why the example is here but new subsection is started at the line 85?

Reply:
This is not clear to me ... ?

Reply:
I mean here that now, after the change, the line 82 contains
`freqtrade backtesting --timerange=20190501-`

It's completely same usage of the option with the examples below, in lines 87-104. But this one is in another subsection since a new subsection starts at the line 85.

The point is do we really need this in two different subsections, here above (line 82) AND below (lines 87-104)?

Reply:
I'm not certain - the first is a quick command "with smaller testset" ... while the next section is advanced usage ... 

but yes, we could also combine the 2 i guess